### PR TITLE
Fix make [golangci|operator-lint]

### PR DESCRIPTION
**Pull request by lpiwowar**

### CODE REVIEW

File: controllers/ansibletest_controller.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (r *AnsibleTestReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	runningJobName := r.GetJobName(instance, externalWorkflowCounter-1)
 	err = r.Client.Get(ctx, client.ObjectKey{Namespace: instance.GetNamespace(), Name: runningJobName}, runningAnsibleJob)
```
Comment (stuggi) [inline controllers/ansibletest_controller.go:111]: is it ok to proceed if `err !=nil` ? should there be a check for if err != nil and maybe special handling for IsNotFound if it is ok to proceed?

File: controllers/common.go [hunk 2]
```diff
@@ -240,7 +241,11 @@ func (r *Reconciler) ReleaseLock(ctx context.Context, instance client.Object) bo
 		},
 	}
 
-	r.Client.Delete(ctx, &cm)
+	err := r.Client.Delete(ctx, &cm)
+	if err != nil && k8s_errors.IsNotFound(err) {
+		return false
+	}
```
Comment (stuggi) [inline controllers/common.go:247]: if there is an error during delete, which is not IsNotFound, is it correct that we return true?

File: controllers/common.go [hunk 3]
```diff
@@ -221,14 +221,15 @@ func (r *Reconciler) AcquireLock(ctx context.Context, instance client.Object, h
 						Namespace: instance.GetNamespace(),
 					},
 				}
-				configmap.EnsureConfigMaps(ctx, h, instance, cms, nil)
-				return true
+
+				err = configmap.EnsureConfigMaps(ctx, h, instance, cms, nil)
+				return err == nil
 			} else {
```
Comment (stuggi) [inline controllers/common.go:227]: could remove the else since the prior if has a return

File: controllers/common.go (hunk 2 above)
Comment (lpiwowar) [reply controllers/common.go]: This should be fixed, thank you!:)

File: controllers/common.go (hunk 3 above)
Comment (lpiwowar) [reply controllers/common.go]: True, thank you!:)

File: controllers/ansibletest_controller.go (hunk 1 above)
Comment (lpiwowar) [reply controllers/ansibletest_controller.go:111]: Yes, better processing of the err would be better here. Thank you! I've added an if-statement that deals with the error. `IsNotFound` error is acceptable in this stage of the control loop as the job gets created in a later stage of the control loop.

**Review by lpiwowar — COMMENTED** [review]
Thank you for the review!

**Comment by stuggi** [comment]
lgtm from the perspective on not knowing the internals of the operator.

**Pull request merged**